projects/RabbitMQ.Client/RabbitMQ.Client.csproj

```diff
@@ -33,6 +33,7 @@
 
   <PropertyGroup>
     <AnalysisMode>AllEnabledByDefault</AnalysisMode>
```

Comment:
This is the most extreme mode that the analyzers can be set to. Only a handful of rules are enabled by default, so this is opting in to all of them.

Reply:
Thanks for this. I'll take a closer look tomorrow and see if I can get it back to the previous behaviour.